src/pages/Recruitment/Recruitment.tsx

```diff
@@ -49,13 +49,15 @@ const Recruitment: React.FC = () =>{
     nextCycle = "January";
   }
   nextYear = (currentCycle === "Fall") ? currentYear+1: currentYear;
+  console.error(`postings: ${postings}`)
```

Comment:
Is this testing code?

Reply:
yup, sorry will remove that! Thanks for catching